avaframe/tests/test_dfa2Aimec.py

```diff
@@ -0,0 +1,62 @@
+''' Tests for dfa2Aimec '''
+
+
+import numpy as np
+import os
+import glob
+
+# Local imports
+import avaframe.ana3AIMEC.dfa2Aimec as dfa2Aimec
+import avaframe.in2Trans.ascUtils as IOf
+from avaframe.in3Utils import cfgUtils
+from avaframe.tests import test_logUtils
+
+
+def test_mainDfa2Aimec(tmp_path):
+
+    # Initialise inputs
+    dirPath = os.path.dirname(__file__)
+    avaTestName = 'avaHockeyChannelPytest'
+    testPath = os.path.join(dirPath, '..', '..', 'benchmarks', avaTestName)
+    pathData = os.path.join(testPath, 'Outputs', 'com1DFA', 'peakFiles')
+    pathDict = dfa2Aimec.mainDfa2Aimec(testPath)
+
+    # get path dictionary for test
+    pathDTest = {}
+    pathDTest['ppr'] = [os.path.join(pathData, 'release1HS_entres_dfa_0.15500_ppr.asc'), os.path.join(pathData, 'release2HS_entres_dfa_0.15500_ppr.asc')]
```

Comment:
```suggestion
    pathDTest['ppr'] = [os.path.join(pathData, 'release1HS_entres_dfa_0.15500_ppr.asc'), 
                                 os.path.join(pathData, 'release2HS_entres_dfa_0.15500_ppr.asc')]
```
same with the others below